From a5e9454cc9f2937ea6d1a187a3195af816083142 Mon Sep 17 00:00:00 2001
From: Tianfang Meng <tianfang@xcdldmz190092.xilinx.com>
Date: Thu, 20 May 2021 19:18:32 +0800
Subject: [PATCH] Vart: don't find python executable

---
 util/src/vaitrace/CMakeLists.txt | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/util/src/vaitrace/CMakeLists.txt b/util/src/vaitrace/CMakeLists.txt
index 26207dc8..92d265b5 100644
--- a/util/src/vaitrace/CMakeLists.txt
+++ b/util/src/vaitrace/CMakeLists.txt
@@ -58,7 +58,8 @@ execute_process(COMMAND ${CMAKE_COMMAND} -E create_symlink
 install(PROGRAMS ${CMAKE_CURRENT_BINARY_DIR}/vaitrace DESTINATION bin)
 
 if(BUILD_PYTHON)
-    find_package(Python3 REQUIRED)
+	#    find_package(Python3 REQUIRED)
+    find_package (Python3 REQUIRED COMPONENTS Development)
     if(IS_EDGE)
         set(VAITRACE_PY_INSTALL_DIR "lib/python${Python3_VERSION_MAJOR}.${Python3_VERSION_MINOR}/site-packages")
     else(IS_EDGE)
-- 
2.17.1

